pygmt/tests/test_plot.py

```diff
@@ -194,6 +196,104 @@ def test_plot_colors_sizes_proj(data, region):
     return fig
 
 
+@check_figures_equal()
+def test_plot_transparency():
+    "Plot the data with a constant transparency"
+    x = np.arange(1, 10)
+    y = np.arange(1, 10)
+
+    fig_ref, fig_test = Figure(), Figure()
+    # Use single-character arguments for the reference image
+    with GMTTempFile() as tmpfile:
+        np.savetxt(tmpfile.name, np.c_[x, y], fmt="%d")
```

Comment:
This [`np.c_`](https://numpy.org/doc/stable/reference/generated/numpy.c_.html) is new to me, but ok.